### PR TITLE
Feature Toggle adjustments

### PR DESCRIPTION
**Pull request by joedakroub**
## Description
This PR does the following:

1. Adjusts the size of the image to match the default icon size in the `SageFeatureToggle`
2. Aligns items to the end of the `aside` container (see mobile screenshots)
3. Removes the `width: 100%` from Switch (does not appear to be needed) so that it can align to the end of the aside container (see mobile screenshots)

## Screenshots
|  Before  |  After  |
|--------|--------|
|Before|After|
|Mobile - Before|Mobile - After|

## Testing in `sage-lib`
1. Visit http://0.0.0.0:4000/pages/component/feature_toggle and ensure that the image and icon sizes match

## Testing in `kajabi-products`
1. (**LOW**) This component is being used in 1 place within the app:
   - [ ] Settings -> Labs

### CODE REVIEW

File: packages/sage-assets/lib/stylesheets/components/_feature_toggle.scss [hunk 1]
```diff
@@ -5,9 +5,9 @@
 ////
 
 
-$-feature-toggle-image-width: rem(120px);
+$-feature-toggle-image-width: rem(96px);
 $-feature-toggle-image-height-min: rem(64px);
-$-feature-toggle-image-height-mobile: rem(120px);
+$-feature-toggle-image-height-mobile: rem(96px);
```
Comment (joedakroub) [inline packages/sage-assets/lib/stylesheets/components/_feature_toggle.scss:10]: Adjusted image size to `96px` to match the icon size.

File: packages/sage-assets/lib/stylesheets/components/_feature_toggle.scss [hunk 2]
```diff
@@ -95,8 +95,9 @@ $-feature-toggle-image-height-mobile: rem(120px);
 }
 
 .sage-feature-toggle__aside {
+  display: flex;
   grid-area: aside;
-  padding-right: sage-spacing(xs);
+  justify-content: flex-end;
```
Comment (joedakroub) [inline packages/sage-assets/lib/stylesheets/components/_feature_toggle.scss:101]: Align items to end of aside container.

File: packages/sage-assets/lib/stylesheets/components/_feature_toggle.scss [hunk 3]
```diff
@@ -132,7 +128,7 @@ $-feature-toggle-image-height-mobile: rem(120px);
 
 .sage-feature-toggle__image {
   align-self: flex-start;
-  max-width: rem(140px);
```
Comment (joedakroub) [inline packages/sage-assets/lib/stylesheets/components/_feature_toggle.scss:135]: This is not needed from what I can see.

File: packages/sage-assets/lib/stylesheets/components/_switch.scss [hunk 4]
```diff
@@ -35,7 +35,6 @@ $-switch-focus-outline-error-color: sage-color(red, 300);
   flex-flow: row wrap;
   align-items: flex-start;
   position: relative;
-  width: 100%;
```
Comment (joedakroub) [inline packages/sage-assets/lib/stylesheets/components/_switch.scss:38]: This is not needed in any context that I tested and is causing it to not align to the end of the aside container in the `FeatureToggle` component.

**Review by pixelflips — APPROVED** [review]
LGTM 👍🏼

**Review by monkeypox8 — APPROVED** [review]
👍🏼 This looks great Joe!

Unrelated to this PR's changes, but I'm wondering if we'd consider applying the truncation for `sage-feature-toggle__title` on > medium breakpoints only. The `white-space: nowrap` from the mixin allows grid blowouts to occur:

<details>
<summary>example on iphone 8</summary>

</details>

**Comment by joedakroub** [comment]
@monkeypox8 That's a great catch! I will add that into this PR. 👍🏼

**Review by philschanely — APPROVED** [review]
Nice clean work here @joedakroub

**Pull request merged**